release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -143,6 +143,83 @@ In earlier versions of {product-title}, Ironic container images used {op-system-
+
+[id="ocp-4-12-agent-based-install-disconnected"]
+==== Deploy an Agent-based {product-title} cluster in a disconnected environment
+You can perform an Agent-based installation in a disconnected environment. To create an image that is used in a disconnected environment, the `imageContentSources` section in the ` install-config.yaml` must contain the mirror information or `registries.conf` file if you are using ZTP manifests. The actual configuration settings to use in these files are supplied by either the `oc adm release mirror` or `oc mirror` command.
+For more information, see xref:../installing/installing_with_agent_based_installer/understanding-disconnected-installation-mirroring.html[Understanding disconnected installation mirroring].
+
+[id="ocp-4-12-agent-based-install-dual-stack"]
+==== Agent-based installation supports single and dual stack networking
+You can create the agent ISO image with the following IP address configurations:
+
+* IPv4
+* IPv6
+* IPv4 and IPv6 in parallel (dual-stack)
+
+For more information, see xref:../installing/installing_with_agent_based_installer/installing-with-agent-based-installer.html#installing-ocp-agent_installing-with-agent-based-installer[Dual and single IP stack clusters].
+
+[id="ocp-4-12-agent-based-install-mce"]
+
+==== The agent deployed {product-title} cluster can be used as a hub cluster
+
+You can install the multicluster engine for Kubernetes(MCE) Operator and deploy a hub cluster with the Agent-based Installer.
+
+// TODO doc PR not merged see xref:..
+
+[id="ocp-4-12-agent-based-install-validations"]
+==== Agent-based installation performs installation validations
+
+The Agent-based Installer performs validations on:
+
+* Installation image generation: The user-provided manifests are checked for validity and compatibility.
+* Installation: The installation service checks the hardware available for installation and emits validation events that can be retrieved with the `openshift-install agent wait-for` subcommands.
+
+For more information, see xref:../installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.html#validations-before-agent-iso-creation_preparing-to-install-with-agent-based-installer[Installation validations].
+
+[id="ocp-4-12-agent-based-install-static-networking"]
+==== Configure static networking in a Agent-based installation
+With the Agent-based {product-title} Installer, you can configure static IP addresses for IPv4, IPv6, or dual-stack (both IPv4 and IPv6) for all the hosts prior to creating the agent ISO image. You can add the static addresses to the `hosts` section of the `agent-config.yaml` file or in the `NMStateConfig.yaml` file if you are using the ZTP manifests.
+Note that the configuration of the addresses must follow the syntax rules for nmstate as described in link:https://nmstate.io/examples.html[NMState state examples].
```

Comment:
Should this lowercase "nmstate" either be marked up `nmstate` or use the "NMState" capitalization?

Reply:
NMState